zokrates_book/src/concepts/stdlib.md

```diff
@@ -1,40 +1,49 @@
 ## Standard library
 
-ZoKrates comes with a number of reusable components. For now, these components are:
+ZoKrates comes with a number of reusable components which are defined at `./stdlib/` in the ZoKrates root folder. The standard library is solely based on the ZoKrates DSL and can be easily extended.
```

Comment:
A user would not have access to the source code, so should it rather be `$ZOKRATES_HOME` for them?